filter.js

```diff
@@ -0,0 +1,12 @@
+const arrString = ((arr) =>{
+  if(Array.isArray(arr)){
+    let ans = arr.filter((x)=> {
```

Comment:
Make your arguments more descriptive, for example, `x` could be `item`, `arr` could be `inputArray`, `ans` could be `filteredArray`

Reply:
Will do that right away